plugins/analytics/public/ivre_api.py

```diff
@@ -0,0 +1,397 @@
+"""Analytics relying on IVRE's data.
```

Comment:
This should probably code in `contrib` instead of `plugins`

Reply:
I think it is similar to Shodan or ProcessIp, that's why I put it here. + IVRE can be installed on any system that has Python 2.6 to 2.7 or 3.3 to 3.7, so I think the dependency should not be a problem. Also, since I had to add new observables that are only used here (for now), I though it would be OK. But of course, this can be discussed, as I said, I'm new to Yeti.

Reply:
`contrib` is meant to receive these externally contributed plugins. No problem with the core modifications that make sense outside of this plugin.

Reply:
All right I understand, particularly your point about maintenance by the core team.